Assets/AppCenter/Editor/AppCenterSettingsMakerAndroid.cs

```diff
@@ -6,8 +6,8 @@
 
 public class AppCenterSettingsMakerAndroid : IAppCenterSettingsMaker
 {
-    private const string AppCenterResourcesFolderPath = "Assets/Plugins/Android/res/values/";
-    private const string AppCenterResourcesPath = AppCenterResourcesFolderPath + "appcenter-settings.xml";
+    public const string AppCenterLoaderResourcesFolderPath = "AppCenterLoaderApp/appcenter-loader/src/main/res/values/";
```

Comment:
It doesn't work. It must not write to this location, it doesn't make any sense if there are no SDK sources in project's location